prompts/utils.py

```diff
@@ -28,7 +29,7 @@ def slugify(value, allow_unicode=False, max_length=50):
 
 def get_unique_path(directory: Path, original_filename, suffix="txt") -> Path:
     filename = original_filename
-    for i in range(1000):
+    for _i in range(1000):
```

Comment:
I've never seen this convention before - why the leading underscore?

Reply:
It's a rust-y convention: if you don't use the variable, prefix it with an underscore.

Reply:
ya, I think the python convention is simply underscore, i.e.

```
for _ in range(1000):
    pass

```

Reply:
Replaced.